python_modules/libraries/dagster-dbt/dagster_dbt/cloud_v2/resources.py

```diff
@@ -20,15 +21,31 @@
 from dagster_dbt.asset_utils import build_dbt_specs
 from dagster_dbt.cloud_v2.client import DbtCloudWorkspaceClient
 from dagster_dbt.cloud_v2.run_handler import DbtCloudJobRunHandler
-from dagster_dbt.cloud_v2.types import DbtCloudJob, DbtCloudWorkspaceData
+from dagster_dbt.cloud_v2.types import (
+    DbtCloudEnvironment,
+    DbtCloudJob,
+    DbtCloudProject,
+    DbtCloudWorkspaceData,
+)
 from dagster_dbt.dagster_dbt_translator import DagsterDbtTranslator
 
 DAGSTER_ADHOC_PREFIX = "DAGSTER_ADHOC_JOB__"
 DBT_CLOUD_RECONSTRUCTION_METADATA_KEY_PREFIX = "__dbt_cloud"
 
 
-def get_dagster_adhoc_job_name(project_id: int, environment_id: int) -> str:
-    return f"{DAGSTER_ADHOC_PREFIX}{project_id}__{environment_id}"
+def clean_adhoc_job_name(name: str) -> str:
```

Comment:
giga nit but I feel like this should just be part of the get_dagster_adhoc_job_name function? Doesn't really feel like it has utility separately?

Reply:
Updated in [d09a7d1](https://github.com/dagster-io/dagster/pull/28552/commits/d09a7d16bd186226fd2139b518d66b0da718ddca)